eng/scripts/Language-Settings.ps1

```diff
@@ -216,3 +216,29 @@ function Update-java-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$nu
 
   Set-Content -Path $pkgJsonLoc -Value $jsonContent
 }
+
+
+# function is used to filter packages to submit to API view tool
+function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName = "")
+{
+  Write-Host "Checking for source jar in artifact path $($artifactDir)"
+  # Find all source jar files in given artifact directory
+  $files = Get-ChildItem "${artifactDir}" | Where-Object -FilterScript {$_.Name.EndsWith("sources.jar")}
+  if (!$files)
+  {
+    Write-Host "$($artifactDir) does not have any package"
+    exit(1)
```

Comment:
It is generally bad form to force exit from a helper function. Instead you should return `$null` and handle at the calling site and exit if that is the entry point script.

Reply:
Fixed